CalibTracker/SiStripHitEfficiency/src/HitEff.cc

```diff
@@ -683,8 +685,8 @@ void HitEff::analyze(const edm::Event& e, const edm::EventSetup& es) {
                     cluster_info.push_back(sqrt(parameters.second.xx()));
                     cluster_info.push_back(parameters.first.y());
                     cluster_info.push_back(sqrt(parameters.second.yy()));
-                    cluster_info.push_back(clusterInfo.signalOverNoise());
-                    //cluster_info.push_back( clusterInfo.getSignalOverNoise() );
+                    cluster_info.push_back(siStripClusterInfo_.signalOverNoise());
+                    //cluster_info.push_back( siStripClusterInfo_.getSignalOverNoise() );
```

Comment:
`getSignalOverNoise()` appears nowhere else in CMSSW. Can we simply remove the commented line?

Reply:
Done. That is better, thanks.